packages/kit/package.json

```
@@ -18,8 +18,7 @@
 	"homepage": "https://svelte.dev",
 	"type": "module",
 	"dependencies": {
-		"@types/cookie": "^0.6.0",
-		"cookie": "^0.6.0",
+		"cookie": "^1.0.2",
```

you suggested on Discord allowing users to use multiple versions to prevent it from being a breaking change. I think that's a reasonable approach (would probably need to tweak the changeset message slightly in that case as well)
```suggestion
		"cookie": "^0.6.0 || ^0.7.0 || ^1.0.2",
```

---

I'm not 100% convinced this would actually prevent this from being a breaking change. If someone ran `npm up` in their project and it grabbed a new version of SvelteKit, it seems likely it would also grab the newest version of `cookie` in the range - which would mean 1.x. Specifying a range here _seems_ better, but I'm not certain which situations it would actually improve.

---

It could be surprising, but at least would give folks a way to opt-out. I'm hoping there aren't many users left using uncommon cookie characters since we've been warning folks about those characters for quite awhile and InLang has updated their library to move away from using a character considered invalid in 1.x.

The bundling route would be cumbersome since we're not already bundling SvelteKit